Module 5

valeron007 opened the pull request:
Problem sort table

valeron007 committed:
added table

valeron007 committed:
sort table

valeron007 committed:
Merge branch 'js-tasks-ru:master' into master

jsru-1 commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

valeron007 committed:
tooltip

valeron007 committed:
new template

jsru-1 commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

valeron007 committed:
review sorttable v2

jsru-1 commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

valeron007 committed:
subelement

jsru-1 commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

valeron007 committed:
sort table

jsru-1 commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

valeron007 committed:
Merge branch 'js-tasks-ru:master' into master

valeron007 committed:
tooltip

valeron007 committed:
s
erge branch 'master' of https://github.com/valeron007/js-20250320_valeriy-andryuschenko

jsru-1 commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

valeron007 committed:
Merge branch 'js-tasks-ru:master' into master

valeron007 committed:
Merge branch 'master' of https://github.com/valeron007/js-20250320_valeriy-andryuschenko

jsru-1 commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

valeron007 committed:
7 module task 1

jsru-1 commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

valeron007 committed:
module 7

jsru-1 commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

valeron007 committed:
change

valeron007 committed:
Merge branch 'master' of github.com:valeron007/js-20250320_valeriy-andryuschenko

valeron007 committed:
module 6-3

jsru-1 commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

ufocoder commented on 05-dom-document-loading/2-sortable-table-v1/index.html:
```diff
@@ -40,7 +40,7 @@
             <img class="sortable-table-image" alt="Image" src="${data[0]?.url || 'https://via.placeholder.com/32'}">
           </div>
-        `;
+        `;it
```
опечатка?

valeron007 committed:
slider

valeron007 committed:
7-1 module

jsru-1 commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

valeron007 committed:
review sorttable

valeron007 committed:
Merge branch 'master' of github.com:valeron007/js-20250320_valeriy-andryuschenko

jsru-1 commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).
